mkdocs/config/config_options.py

```diff
@@ -382,6 +382,16 @@ def post_validation(self, config, key_name):
 
         config[key_name] = extras
 
+        if not extras:
+            return
+
+        self.warnings.append((
+            'The following files have been automatically included in the '
+            'documentation build and will be added to the HTML: {}. In '
+            'MkDocs they will need to be explicitly included with the '
```

Comment:
Shouldn't that be: " In MkDocs version 1.0 they will need to be explicitly included ..."?

Notice that I added "version 1.0".